docs/baking.md

```diff
@@ -30,7 +30,8 @@ that you can use by following the [installation instructions](https://www.raspbe
 In order to run a baking instance, you'll need the following Tezos binaries:
 `tezos-client`, `tezos-node`, `tezos-baker-<proto>`, `tezos-endorser-<proto>`.
 
-The currently supported protocol is `012-Psithaca` (used on `ithacanet` and `mainnet`).
+The currently supported protocol is `012-Psithaca` (used on `ithacanet` and `mainnet`) and
+`013-PtJakart` (jakartanet).
```

Comment:
```suggestion
The currently supported protocols are `012-Psithaca` (used on `ithacanet` and `mainnet`) and
`013-PtJakart` (used on `jakartanet`).
```

Reply:
@rvem I un-resolved this because it still applies after some fixups (there is a `ised` typo and it's not using the plural)